pkg/driver/node/mounter/pod_mounter_test.go

```diff
@@ -234,6 +234,60 @@ func TestPodMounter(t *testing.T) {
 			assert.Equals(t, credentialprovider.CredentialDirPerm, credDirInfo.Mode().Perm())
 		})
 
+		t.Run("success: driver environment s3 endpoint url", func(t *testing.T) {
```

Comment:
Shall we also test invalid endpoints? What happens if I provide a wrong string? Maybe it's there already but as we ad support for endpoint, I was expecting some kind of validation

Reply:
For now this is indeed checked at mountpoint-s3 level.
Ideally I would want the driver to not start if S3 endpoint, AK or SK is not provided. I created a ticket for this, it was in my to do list, I need the same for AK, SK at driver level. Will add validation for all 3 in that ticket as well
Ticket reference: https://scality.atlassian.net/browse/S3CSI-14

Reply:
Thanks!